### DISCUSSION TITLE
Restart procedure?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1033] [Q&A] [answered]
Hello,

The page for the restart procedure hasn't been updated for a while, is this still current?
https://github.com/ufs-community/ufs-weather-model/wiki/Restarting-the-Coupled-Model

If not, could someone provide me with the updated restart procedure?

**Comment by DeniseWorthen** [comment, marked as answer]
There has been no update that I know of to the restart procedure.
